app/routes/members-area/route.module.scss

```diff
@@ -0,0 +1,10 @@
+.page {
+    padding-top: 100px;
+    width: 100%;
```

Comment:
Is `width: 100%` necessary? If this has to be added to every page, then it would make more sense to remove the container style that prevents automatic stretching.

Reply:
Ah. Is it because of `margin: 0 auto`?